src/renderer/Renderer.ts

```diff
@@ -130,65 +128,78 @@ abstract class Renderer {
     return this._panels[index] || null;
   }
 
+  /**
+   * Update all panel sizes
+   * @ko 모든 패널의 크기를 업데이트합니다
+   * @chainable
+   * @return {this}
+   */
+  public updatePanelSize(): this {
+    this._panels.forEach(panel => panel.resize());
+    return this;
+  }
+
   /**
    * Insert new panels at given index
    * This will increase index of panels after by the number of panels added
    * @ko 주어진 인덱스에 새로운 패널들을 추가합니다
    * 해당 인덱스보다 같거나 큰 인덱스를 가진 기존 패널들은 추가한 패널의 개수만큼 인덱스가 증가합니다.
    * @param {number} index Index to insert new panels at<ko>새로 패널들을 추가할 인덱스</ko>
-   * @param {Flicking.ElementLike | Flicking.ElementLike[]} element A new HTMLElement, a outerHTML of element, or an array of both
-   * <ko>새로운 HTMLElement, 혹은 엘리먼트의 outerHTML, 혹은 그것들의 배열</ko>
+   * @param {any[]} elements An array of element or framework component with element in it<ko>엘리먼트의 배열 혹은 프레임워크에서 엘리먼트를 포함한 컴포넌트들의 배열</ko>
    * @return {Panel[]} An array of prepended panels<ko>추가된 패널들의 배열</ko>
    */
-  public insert(index: number, element: ElementLike | ElementLike[]): Panel[] {
+  public batchInsert(...items: Array<{
+    index: number;
+    elements: any;
```

Comment:
I think it would be better to infer it with the any[] type.

Reply:
Thanks for checking that, I'll fix it.

Reply:
It would be nice if an explanation of what polyfill to add was added.